back-end/dao/intentdao.js

```diff
@@ -17,6 +17,12 @@ export default class IntentDao extends IntentInterface{
   
   async postIntents(content, override) {
     try {
+      if (override) {
+        // Delete previous intents with same project id
+        const projectId = content[0].project_id
```

Comment:
Won't this over look the case when a transcript has more than one project id in it?

Reply:
I'm assuming each transcript should only have one project id (since it should only be one bot). I'm not sure if that's right but if not I can quickly fix it.